src/resources/elements/primeDesignSystem/ptooltip/ptooltip.ts

```diff
@@ -11,11 +14,15 @@ export class PTooltip {
   @bindable.boolean visible = true;
   @bindable.boolean allowHtml = false;
   @bindable appendTo: Props["appendTo"] | string;
+  @bindable.number maxWidth: number = MAX_WIDTH;
```

Comment:
why did you make this a bindable?  We have no current use for it.

Reply:
ok will revert

cc @IonelLupu

Reply:
I was thinking this is needed in case we will have a tooltip that will exceed this size in the future. Just to be safe. @dkent600

Reply:
Generally best practice is not to code for possible future cases unless we are confident they will be needed.  @IonelLupu